### ISSUE TITLE
what is the point of having a free subscription ?

### ISSUE THREAD

**Issue by lhapaipai** [dev-xo/remix-saas#350]
Good evening again,
I would have liked to write this in a discussion section because it is not really a issue.

In fact I was wondering what was the point of creating a monthly and annual subscription **with Stripe** for the free plan with a 0€ rate?
This adds code to maintain and I would be curious to understand what pushed you to make this decision?

Thanks in advance!

**Comment by dev-xo** [comment]
Hello @lhapaipai.

The original code comes from a previous Remix Stack called Stripe Stack. As far as I remember, the idea of having a free subscription attached to the user was to also allow the user to have a Free Trial.

Nowadays, I prefer to create the customer at the moment they are about to subscribe, but in the end, it’s simply two approaches, and both could work.

I have plans for a major refactor of Remix SaaS with the release of Remix v3, and that may change to create the customer only when required, without attaching a free subscription to it.

---

In summary, I think the intent was to easily allow anyone to attach a trial to that free subscription, but that’s something I do not do myself nowadays, and as you said, it’s more code to maintain.

Still an option, and it works as expected, though.

**Comment by lhapaipai** [comment]
thank you @dev-xo for your answer !
